types/mongoose-paginate-v2/package.json

```diff
@@ -1,6 +1,7 @@
 {
     "private": true,
     "dependencies": {
-        "mongoose": "^6.0.13"
+        "mongodb": "3.6.8",
+        "mongoose": "^5.13.2"
```

Comment:
I have pinned the `mongoose` version to the same one used in the [original package](https://github.com/aravindnc/mongoose-paginate-v2/blob/master/package.json#L85).
I have also pinned the `mongodb` accordingly to the version used by `mongoose@5.13.2`.

This introduces the changes concerning `CollationDocument`, `EnforceDocument` in `index.js`.